### ISSUE TITLE
Consider replacing the hero illustration by the try me search bar

### ISSUE THREAD

**Issue by gmourier** [meilisearch/landing#512]
![Capture d’écran 2022-10-12 à 21 15 58](https://user-images.githubusercontent.com/3692335/195428911-9c1909f1-fdd6-4f06-9362-ea8dd9bf1e57.png)

One of our community users reached out to me to say: "That should be the first thing people see. Promote it [...] On my Mac, I never think of scrolling down below the fold, typically." 

I agree with him. The aha moment is instantaneous, more visible, and impactful.

**Comment by irevoire** [comment]
Yeah, I never told it, but every time I send this page to my friends, they try to type in this bar and then miss the second one; thus, it could be a good idea :+1:

**Comment by mdubus** [comment]
Thanks for the suggestion 🙏 We'll have to wait for @lpinot to come back before taking any decision 😀

**Comment by lpinot** [comment]
Hey @gmourier I did a test [here](https://www.figma.com/file/NOR5BQg1ITjtzFfj1jPdpc/Homepage?node-id=2806%3A9372&t=lzTTFL5sEAe4ULfT-1) and I think that is a great idea, we should update the landing with that new layout when we have some time! 
<img width="830" alt="Screenshot 2022-11-14 at 15 02 11" src="https://user-images.githubusercontent.com/83015630/201679668-98728e10-8a04-4e2d-8f6b-c4068f282f1c.png">

**Comment by gmourier** [comment]
Thanks for the mock-up and the following, @lpinot!

**Comment by irevoire** [comment]
Oh it look so nice with all the little stars :pleading_face:
